### DISCUSSION TITLE
Hypothetically, can forward nodes run Security Onion 2.3 while the management node runs Security Onion 14.04?

### DISCUSSION THREAD

**Discussion by wem0019** [Security-Onion-Solutions/securityonion#3390] [Unsupported Versions] [answered]
If this is possible, what information and capabilities are lost when running such a distributed system?

**Comment by zaphode** [comment]
I don't think you can run 2.3 Forward nodes without a 2.3 Management box somewhere. No Salt on 14.04. I think the docs say you need a 2.3 Management node.

**Reply by wem0019** [reply]
Thank you! That's what I figured, but I just needed confirmation. I appreciate your help!

**Comment by weslambert** [comment, marked as answer]
Can confirm this is not recommended, or supported.  Your nodes should all be on the same version.  You may consider setting up a new manager and then cutting over sensors to avoid shutting off all visibility at once.  You can also upgrade 16..04 nodes to 2.x.  You could try upgrading 14.04->-16.04->2.x, however your time would probably be best spent with a fresh install.

**Reply by wem0019** [reply]
Thank you! That's what I figured, but I just needed confirmation. I appreciate your help!
